.changeset/quick-onions-leave.md

```diff
@@ -0,0 +1,33 @@
+---
+'astro': patch
+---
+
+Implements legacy content and data collections using `glob()` loader
```

Comment:
```suggestion
Refactors legacy `content` and `data` collections to use the Content Layer API `glob()` loader for better performance and to support backwards compatibility
```
Whatever the actual reasons/benefits are: preparing for easier transitions from old to new? I like the use of "refactors" here rather than "implements" because it helps with the "this is an old/existing thing that we've changed" messaging which is how *they* experience this.